### DISCUSSION TITLE
Problems indexing folder structure

### DISCUSSION THREAD

**Discussion by samarkanda03** [bpatrik/pigallery2#391] [Q&A]
Hi,

I'm installing pigallery2 on a Raspberry pi 4 using Docker. Firt's of all, congratulations, the software is fantastic!

I have noticed that pigallery hasn't build correctly the directory tree in some cases. I have a large library with photos from 20 years ago so i think that maybe there are problems in some folders.

My question is: there is some log that i could read to search errors?

Thanks in advance.

PD: sorry for my poor english T_T

**Comment by samarkanda03** [comment]
Well, i am reviewing docker logs and i have found a lot of related to photo metadata

sqlMessage: "Out of range value for column 'metadataCreationdate' at row 1",
The value that pigallery is trying to insert is -2211753600000

This prevent the image to be inserted in database so it dissapear from the gallery.

Is there any way to avoid pigallery inspect metadata? or is it posible to set a default value to this column when the scanner detects an error?

Thanks in advance

**Comment by bpatrik** [comment]
There is not solution for that at the moment. 

If you are using MySQL, I recommend trying sqlite that is less sensitive for such things or try manually changing the SQL scheme after it get created to enable inserting -2211753600000.

Plus you can send one of your photos here and I will add it to the unit tests to make sure it works with the app (no deadline promises, hopefully during the holiday, I will have time for this project)
